src/main/java/fly/wild/wizards/serverside/ServerConnector.java

```diff
@@ -0,0 +1,328 @@
+package fly.wild.wizards.serverside;
+
+import java.io.IOException;
+import java.time.LocalTime;
+import java.util.UUID;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.dmr.ModelNode;
+import org.jboss.as.cli.ControllerAddress;
+import org.jboss.as.cli.Util;
+import org.jboss.as.cli.impl.ModelControllerClientFactory;
+import org.jboss.as.cli.operation.OperationFormatException;
+import org.jboss.as.cli.operation.impl.DefaultOperationRequestBuilder;
+
+import fly.wild.wizards.tlswizard.controller.OneWayTLSConfigurationConfiguration;
+import fly.wild.wizards.tlswizard.controller.TLSConfiguration;
+
+public class ServerConnector {
+
+	private String ipAddress;	
+	private String distinguishedName;
+	private String alias;
+	private StringBuilder logMessage;
+	private final String STRING_CLEAR_TEXT_VALUE_KEY_STORE = "keystorepass";
+	private final String STRING_ALGORITHM_VALUE = "RSA";
+	private final int INT_KEY_SIZE_VALUE = 2048;
+	private final int INT_VALIDITY_VALUE = 365;
+	private final String STRING_SERVER_VALUE = "default-server";
```

Comment:
This one also, may be we might need these values for configuring two-way TLS. In case the method for it is different from the one for one-way TLS, it might be good to define this here. WDYT?